src/ApiClient/models/InternalComms/index.ts

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable import/no-cycle */
```

Comment:
Why, unless there's a really really really good reason for this, I don't want any circular dependencies anywhere. They have burned me once, it was not fun to debug, never again.

Reply:
well _technically_ this shouldn't cause any breaking circularness, and if so, it's only within auth <-> error comms

Reply:
I would still rather avoid them